windows/README.md

```diff
@@ -37,4 +37,19 @@ Get-Service -Name "SF*" | foreach {sc.exe config $_.Name start= disabled}
 
 ```
 Get-Service -Name "SF*" | foreach {sc.exe start $_.Name}
+```
+
+
+# Windows Command Line Scripts
+
+### Add extra SmartFace service
+
+#### Add extra Camera service
+```
+SC CREATE "SFCam11" binpath= "C:\Program Files\Innovatrics\SmartFace\SmartFace.Camera.exe  --serviceName SFCam11" DisplayName= "SmartFace Camera 11" start= delayed-auto
```

Comment:
These commands only work in `CMD` or `Powershell Core`
on Classic Powershell (at least on my machine, version `5.1.22000.282`)  `SC` is an alias for cmdlet `Set-Content` so it won't work

either put a note here, or replace `SC` with `sc.exe`